en/docs/includes/analytics/configure-synapse-gateway.md

```diff
@@ -13,7 +13,7 @@ Follow the instructions below to publish analytics data to the analytics cloud v
       <div class="admonition note">
       <p class="admonition-title">Note</p>
       <ul><li><p>This is the basic configuration that you need to publish analytics data to the analytics cloud.</p></li>
-      <li>If you need to change the following default values of the Worker Thread Count, Queue Size or Client Flushing Delay, see the <a href="https://apim.docs.wso2.com/en/4.2.0/api-analytics/gateways/configure-synapse-gateway/#advanced-configurations">Advanced configurations</a>.
+      <li>If you need to change the following default values of the Worker Thread Count, Queue Size or Client Flushing Delay, see the [Advanced configurations](/api-analytics/choreo-analytics/getting-started-guide).  .
```

Comment:
Should we add {{base_path}} here too? Let's check if this link is rendered properly.